docs/components/cursor.md

```diff
@@ -183,3 +183,16 @@ pick up event with the `begin` attribute:
 
 To play with an example of a cursor with visual feedback, check out the [Cursor
 with Visual Feedback example on CodePen][cursor-codepen].
+
+## XR Select Cursor
+
+When an XR `"selectstart"` event happens the raycaster picks an element based upon it's current location.
+This works for handheld AR, and headmounted VR and AR. This works well with the mouse `rayOrigin` too.
+
+```html
+<a-scene
+  cursor__mouse="rayOrigin: mouse"
```

Comment:
multiple components for this might feel like an overkill. wondering if we can simplify somehow. maybe `rayOrigin: mouse` also considers selectstart? Not a fan either because makes `rayOrigin: mouse` feel deceiving (not really only about mouse anymore)

Reply:
I can think of a few situations in which a developer may prefer to handle all of their own XR select behaviours but still take advantage of the mouse cursor.

[My XR Input Handling library](https://github.com/AdaRoseCannon/handy-work/blob/main/README-AFRAME.md) in particular makes it really simple to start building your x-platform own XR cursor using the `entity` method of the cursor component component.

Reply:
Any other ideas to avoid the multiple component approach?

Reply:
have rayOrigin be an array of strings?

Reply:
array would weird too I think. probably even weirder. Let's go with the multiple components for now but I don't feel super comfortable. Thanks for the patience